internal/containerizer/dockerfilecontainerizer.go

```diff
@@ -182,6 +183,19 @@ func (d *DockerfileContainerizer) GetContainer(plan plantypes.Plan, service plan
 			container.AddExposedPort(portToExpose)
 		}
 
+		// is "ports" present ?
+		if val, ok := segmentRecord["ports"]; ok {
+			switch reflect.TypeOf(val).Kind() {
```

Comment:
Since it is only one type check, would it be better to use `if` instead of switch case?

Would it be better to use a check like val.([]int) to get this value, so that it does both the type check and also get the value in the right format in one go?